src/stores/routePathStore.ts

```diff
@@ -1,18 +1,14 @@
 import { action, computed, observable } from 'mobx';
 import _ from 'lodash';
-import { IRoutePath, IRoutePathLink, INode } from '~/models';
+import { IRoutePath, IRoutePathLink } from '~/models';
 import lengthCalculator from '~/util/lengthCalculator';
 import { validateRoutePathLinks } from '~/util/geomValidator';
 import UndoStore from '~/stores/undoStore';
 
-export enum AddRoutePathLinkState {
-    SetTargetLocation,
-    AddLinks,
-}
-
-export enum AddLinkDirection {
-    BeforeNode,
-    AfterNode,
+// Is the neighbor to add either startNode or endNode
+export enum NeighborToAddType {
+    StartNode,
```

Comment:
In my opinion the old beforeNode and afterNode is more declarative. But we could figure out a third option also, since I don't really understand BeforeNode and AfterNode at the moment

Reply:
I think as a nodeType (NeighborToAddType) StartNode / EndNode fits better because in our ILink we have startNode & endNode so I would go with this naming